assets/question_sets/strings.js~

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
```

Comment:
Why include this file? (Make sure to look at the "Files Changed" tab on https://github.com/google/tie/pull/147/files and ensure that the change is what you expect it to be. You can also use tools like "git diff" or "meld".)